src/util/message.cpp

```diff
@@ -40,6 +40,7 @@ static constexpr unsigned int BIP322_REQUIRED_FLAGS =
 |   SCRIPT_VERIFY_CLEANSTACK
 |   SCRIPT_VERIFY_P2SH
 |   SCRIPT_VERIFY_WITNESS
+|   SCRIPT_VERIFY_TAPROOT
```

Comment:
In e100fed327a0bd3ce11a882782c6cce3487fb28a "fix: add missing SCRIPT_VERIFY_TAPROOT to BIP322_REQUIRED_FLAGS"

Could be squashed into 30642a185971555ea68bd5b80f87d633ddd09593 "message: add BIP-322 verification support (without POF)"